### PR TITLE
Add initial changes and wifi setting script.

### PR DESCRIPTION
**Pull request by liayoo**

### CODE REVIEW

**Comment by platfowner** [comment]
question: is this a repo migration? @liayoo 
if so, is the commit history reserved?

**Comment by liayoo** [comment]
> question: is this a repo migration? @liayoo if so, is the commit history reserved?

Yes, this is migration, but since we only want public code here (no firebase-admin related), I thought it might be better to copy only what we want instead of migrating the entire history & repo.

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # miniegg-plus-device
-MiniEgg is Comcomm's own NFT that can be hatched into other NFTs. MiniEgg Plus is a project that interacts with the Blockchain to allow this Miniegg NFT to grow through games.
+MiniEgg is Comcomm's own NFT that can be hatched into other NFTs. MiniEgg+ is a project that interacts with the Blockchain to allow this Miniegg NFT to grow through games. MiniEgg+ is currently a [RP2040 (Raspberry Pi Pico)](https://www.raspberrypi.com/products/raspberry-pi-pico/), but you can upgrade the spec if you'd like.
 
 ## Prerequisite
 - Node version: v14.17.6+
 - NPM version: v6.14.15+
+- MiniEgg+ version: v1.1.0
+- Kaluma version: [v1.0.0](https://github.com/kaluma-project/kaluma/releases/tag/1.0.0)
```
Comment (ehgmsdk20) [inline README.md]: [v1.0.0-beta.13](https://github.com/kaluma-project/kaluma/releases/tag/1.0.0-beta.13) was used, not kaluma v1.0.0.

Comment (ehgmsdk20) [reply README.md]: The firmware file name also seems to need to be changed from upgrade-kaluma-rp2-pico-1.0.0.uf2 to upgrade-kaluma-rp2-pico-1.0.0-beta.13uf2.

Comment (liayoo) [reply README.md]: thanks for letting me know! updated.

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+const path = require('path');
+const webpack = require('webpack');
+
+const isProduction = process.env.NODE_ENV == 'production';
+const filePath = process.env.FILE_PATH || 'src/index.js';
```
Comment (ehgmsdk20) [inline webpack.config.js:5]: 'src/index.js' -> 'src/settings/assets.js' or 'src/settings/reset-wifi.js'? This is the file that is built by default.

Comment (liayoo) [reply webpack.config.js:5]: Wouldn't this change the behavior though? Plus we still need to build src/index.js after resetting the wifi and will need it in the future for other things as well, so wouldn't it make sense to keep index.js as the default value? Or another solution can be removing the default value and always requiring FILE_PATH.

Comment (ehgmsdk20) [reply webpack.config.js:5]: If the index.js file is going to be added later, I think you can leave it as it is for now.

Comment (liayoo) [reply webpack.config.js:5]: The file is already there~

**Comment by liayoo** [comment]
@ehgmsdk20 could you let me know if this script works on your device?

File: README.md [hunk 3]
```diff
@@ -1 +1,52 @@
...
+```
+
+The default build target file is 'src/index.js'. If you want to build other file,
+```
+FILE_PATH=<file_path> npm run build
+```
+
+### Upload scripts
+```
+sh upload.sh {local|dev|prod} [<filePath>]
```
Comment (ehgmsdk20) [inline README.md]: scripts/upload.sh?

**Review by ehgmsdk20 — APPROVED** [review]
I checked that all scripts and files are working normally.
LGTM with minor comments! Thanks!

File: README.md (hunk 3 above)
Comment (liayoo) [reply README.md]: done, thanks!

**Comment by liayoo** [comment]
Thanks for the reviews! Now merging~

**Pull request merged**